pkgs/applications/misc/far2l/default.nix

```diff
@@ -77,9 +67,9 @@ stdenv.mkDerivation rec {
   stripDebugList = [ "bin" "share" ];
 
   meta = with lib; {
-    description = "An orthodox file manager";
+    description = "Linux port of FAR Manager v2, a program for managing files and archives in Windows operating systems";
     homepage = "https://github.com/elfmz/far2l";
-    license = licenses.gpl2;
+    license = licenses.gpl2Plus; # NOTE: might change in far2l repo soon, check next time
     maintainers = [ maintainers.volth ];
```

Comment:
Can you add yourself here as a maintainer?

Reply:
Should I? :) Merely noticed that the software is not workable on NixOS without GUI and contacted far2l folks and ruled out all I have to do is update. Maybe it's a build too bold to sign up as a maintainer. What is the policy?

Reply:
> What is the policy?

AFAIK, there is no policy. Are you interested in seeing updates in this package and have some context on how the package works? You can add yourself as a maintainer.

The only thing it really changes is that you will receive a notification every time the package is update by someone that is not you, so you can see the changes and review them.

Reply:
OK then )